### PR TITLE
Updating systest readme and Makefile defaults

### PR DESCRIPTION
**Pull request by jonZlotnik**
## Motivation

Closes #

## Changes

- Fixing broken links
- Filling in missing readme steps

## Test Plan

Manually ran the systests and watched them pass.

## DevOps Notes

- [ ] This PR does not require configuration changes (e.g., environment variables, GitHub secrets, VM resources)
- [x] This PR does not affect public APIs
- [x] This PR does not rely on a new version of external services (PoET, elasticsearch, etc.)
- [x] This PR does not make changes to log messages (which monitoring infrastructure may rely on)

### CODE REVIEW

File: systest/Makefile [hunk 1]
```diff
@@ -7,7 +7,7 @@ clusters ?= 1
 size ?= 10
 level ?= debug
 bootstrap ?= 5m
-node_selector ?= cloud.google.com/gke-nodepool=dmitry-ci-preemtible-1
```
Comment (countvonzero) [inline systest/Makefile:10]: your branch is not the latest version. i have updated this a while back.

Comment (jonZlotnik) [reply systest/Makefile:10]: Yeah, I got mucho confused when I created the PR and there were conflicts.

Comment (jonZlotnik) [reply systest/Makefile:10]: Fixing...

Comment (jonZlotnik) [reply systest/Makefile:10]: Oh boy, this was totally fixed and I just didn't see the updates. Killing this PR

**Pull request closed without merging**